tests/accounts/test_integration.py

```diff
@@ -225,6 +225,25 @@ def test_account_is_marked_as_canceled(self):
         self.assertEqual(self.retrieved_account['status'], 'Canceled')
 
 
+class TestSetClientIDForAccount(AccountTestCase):
+
+    def setUp(self):
+        service = SitewitService()
+        self.created_account = self.create_account()
+
+        self.new_client_id = uuid.uuid4()
```

Comment:
yes, it works now as hex. I would like to follow docs, but it is unlikely this will be changed by SW.